simtools/model/mirrors.py

```diff
@@ -123,34 +123,53 @@ def _read_mirror_list_from_ecsv(self):
     def _read_mirror_list_from_sim_telarray(self):
         """
         Read the mirror list in sim_telarray format and store the data.
+        Allow to read mirror lists with different number of columns.
```

Comment:
Do we not require the mirror list to be always same shape with a schema?

Reply:
The schemas will come later. We plan to have all those tables in astropy.tables, which will make the usage/implementation of the schemas much easier. For now, we don't apply any schemas .

Reply:
Okey